groupie/src/test/java/com/genius/groupie/SectionTest.java

```diff
@@ -565,4 +583,22 @@ public void insertGroupToNestedSectionNotifiesAtCorrectIndex() throws Exception
         rootSection.add(1, nestedSection3);
         verify(groupAdapter).onItemRangeInserted(rootSection, 2, 2);
     }
+
+    @Test
+    public void removeGroupFromNestedSectionNotifiesAtCorrectIndex() throws Exception {
+        final Section rootSection = new Section();
+
+        rootSection.setGroupDataObserver(groupAdapter);
```

Comment:
Hm ... I guess it's fine. I was confused by it, but you're right, it makes no difference.